src/formatHit.test.js

```diff
@@ -36,6 +36,23 @@ describe('formatHit', () => {
       }
     }),
     getTestCase({
+      name: 'locale_names[1] is matching',
```

Comment:
OK, I had a local diff to target the dev cluster and it was stopping the test suite before reaching the test :/